data/beaconrestapi/src/main/java/tech/pegasys/artemis/beaconrestapi/networkhandlers/ENRHandler.java

```diff
@@ -0,0 +1,35 @@
+/*
+ * Copyright 2019 ConsenSys AG.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+ * an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations under the License.
+ */
+
+package tech.pegasys.artemis.beaconrestapi.networkhandlers;
+
+import tech.pegasys.artemis.beaconrestapi.handlerinterfaces.BeaconRestApiHandler;
+
+// TODO: remove unused variable when discovery is integrated
+@SuppressWarnings("UnusedVariable")
```

Comment:
Probably don't need this warning now as I can't see any unused variables.

Reply:
done